packages/backend/src/api/routes/profile.ts

```diff
@@ -26,30 +27,38 @@ export const initProfileRoutes = (
     }),
   );
 
+  router.get(
+    apiPath(path, ProfileApiRoutes.GET_FULL_USER_DATA),
+    wrap(async (req: Request) => {
+      const { userId } = req.params;
```

Comment:
SECURITY!!! Get userId from token. This method is allowed only for authenticated user

Reply:
yup, my bad